peps/pep-0562.rst

```diff
@@ -10,6 +10,9 @@ Post-History: 09-Sep-2017
 Resolution: https://mail.python.org/pipermail/python-dev/2017-December/151033.html
 
 
+.. canonical-doc:: `Customizing Module Attribute Access <https://docs.python.org/3.13/reference/datamodel.html#customizing-module-attribute-access>`_
```

Comment:
Why to pick a specific version?

```suggestion
.. canonical-doc:: `Customizing Module Attribute Access <https://docs.python.org/3/reference/datamodel.html#customizing-module-attribute-access>`_
```

BTW, other links use labels.  Why not add a label for this section first?